src/API_modules/API_functionality.js

```diff
@@ -0,0 +1,40 @@
+import { key } from './API_key.js';
+
+export default class API {
+  constructor() {
+    this.apiName = 'Haider Ahmad';
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  addScores = async (playerName, playerSocre) => {
+    const response = await fetch(
+      `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${key}/scores/`,
+      {
+        method: 'POST',
+        headers: {
+          'Content-type': 'application/json; charset=UTF-8',
+        },
+        body: JSON.stringify({
+          user: playerName,
+          score: playerSocre,
+        }),
+      },
+    );
+    const responseJson = await response.json();
+    return responseJson;
+  };
+
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
- [ ] I noticed that you disabled **Eslint** . Please remove the **eslint disable** statement and try to fix all the errors you are getting. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  :+1: